include/core-image-renesas-base.inc

```diff
@@ -35,7 +35,148 @@ IMAGE_BOOT_FILES:rzg2l-sbc = " \
+
+DEPENDS += " firmware-pack"
+
+# Install core-image-qt.env to host/env
+python do_rootfs_wicenv:append () {
+    target_dir = os.path.join(depdir, 'host/env')
+    os.makedirs(target_dir, exist_ok=True)
+    bb.utils.copyfile(os.path.join(outdir, basename) + '.env', os.path.join(target_dir, basename) + '.env')
+}
+
+# Install file rootfs.manifest to host/build
+python write_image_manifest:append () {
+    # Create target dir host/build
+    target_dir = os.path.join(deploy_dir, 'host/build')
+    os.makedirs(target_dir, exist_ok=True)
+
+    # Copy the .manifest file to the deploy_dir/host/build folder
+    target_manifest = os.path.join(target_dir, os.path.basename(manifest_name))
+    os.system(f"cp {manifest_name} {target_manifest}")
+
+    # Create a symbolic link to the new manifest file in the deploy directory
+    if os.path.exists(target_manifest) and link_name:
+        target_manifest_link = os.path.join(target_dir, link_name + ".manifest")
+        if target_manifest_link != target_manifest:
+            if os.path.lexists(target_manifest_link):
+                os.remove(target_manifest_link)
+            os.symlink(os.path.basename(target_manifest), target_manifest_link)
+}
+
+# Install file testdata.json to host/build
+python write_image_test_data:append () {
+    # Create the directory deploy_dir/host/build
+    target_dir = os.path.join(deploy_dir, 'host/build')
+    os.makedirs(target_dir, exist_ok=True)
+
+    # Copy the .testdata.json file to deploy_dir/host/build
+    target_testdata = os.path.join(target_dir, os.path.basename(testdata_name))
+    os.system(f"cp {testdata_name} {target_testdata}")
+
+    # Create a symbolic link in deploy_dir pointing to the new file in deploy_dir/host/build
+    if os.path.exists(target_testdata) and link_name:
+        target_testdata_link = os.path.join(target_dir, "%s.testdata.json" % link_name)
+    if os.path.lexists(target_testdata_link):
+        os.remove(target_testdata_link)
+    os.symlink(os.path.basename(target_testdata), target_testdata_link)
+}
+
+# Move wic file from deploy complete folder to target/images directory
+IMAGE_CMD:wic:prepend() {
+    target_dir="${IMGDEPLOYDIR}/target/images"
+
+    # Create the target directory if it does not exist
+    mkdir -p "${target_dir}"
+
+    # Check if the build artifact for the wic image exists; if not, copy it to the build directory
+    if [ ! -f "${DEPLOY_DIR_IMAGE}/Image" ]; then
+        cp "${DEPLOY_DIR_IMAGE}/target/images/Image" "${DEPLOY_DIR_IMAGE}/"
+    fi
+
+    if [ ! -f "${DEPLOY_DIR_IMAGE}/rzpi.dtb" ]; then
+        cp "${DEPLOY_DIR_IMAGE}/target/images/dtbs/rzpi.dtb" "${DEPLOY_DIR_IMAGE}/"
+    fi
+}
+
+IMAGE_CMD:wic:append() {
+    # Remove existing wic file and move the output to target directory
+    rm -f "${target_dir}"/*.wic
+    mv "${out}.wic" "${target_dir}/${IMAGE_LINK_NAME}.wic"
+     cp "${target_dir}/${IMAGE_LINK_NAME}.wic" "${IMGDEPLOYDIR}/${IMAGE_NAME}.wic"
```

Comment:
Check the indentation

Reply:
I wonder if we're using copy command instead of mv, could resolve the error when the wic file is missing from `IMGDEPLOYDIR`. Please try and confirm
```
cp "${out}.wic" "${target_dir}/${IMAGE_LINK_NAME}.wic"
```

so then we do not need to use 

```
cp "${target_dir}/${IMAGE_LINK_NAME}.wic" "${IMGDEPLOYDIR}/${IMAGE_NAME}.wic"
```